test/common/config/grpc_stream_test.cc

```diff
@@ -4,10 +4,11 @@
 #include "common/protobuf/protobuf.h"
 
 #include "test/common/stats/stat_test_utility.h"
+#include "test/mocks/common.h"
 #include "test/mocks/config/mocks.h"
 #include "test/mocks/event/mocks.h"
 #include "test/mocks/grpc/mocks.h"
-#include "test/mocks/upstream/mocks.h"
+#include "test/mocks/runtime/mocks.h"
```

Comment:
Why is runtime mocks needed here? I don't see snapshot/loader/overridelayer in this file.

Reply:
When I started run the refactoring scripts, this file previously used Runtime::MockRandomGenerator so it needs runtime mocks. And it includes runtime/mocks.h from upstream/mocks.h. So after removing upstream mocks my scripts added runtime. Now random has been moved out from runtime(#11960). 
Thanks for pointing out, I will fix those over-inclusion soon